slangpy/tests/slangpy_tests/helpers.py

```diff
@@ -82,6 +80,17 @@ def get_device(
             "Please set use_cache=False if you want to use existing_device_handles."
         )
 
+    selected_adaptor_luid = None
```

Comment:
slightly different then in `sglhelpers.py`, maybe worth consolidating. i.e. we could maybe select the adapter in conftest.py?